src/jit/gentree.h

```diff
@@ -2463,10 +2471,34 @@ struct GenTreeCall final : public GenTree
 
     regMaskTP         gtCallRegUsedMask;      // mask of registers used to pass parameters
 
+    // For now Return Type Descriptor is enabled only for x64 unix.
+    // TODO: enable for all arch that support multi-reg returns (i.e. arm64/arm32/x86)
```

Comment:
I believe that coding conventions require all TODOs to be qualified. I think that there are both TODO-X86 and TODO-ARM as well as TODO-Cleanup items (the latter for targets that are currently fully functional but that we want to unify).